corehq/apps/cloudcare/views.py

```diff
@@ -117,7 +117,11 @@ def get(self, request, domain, urlPath):
                 apps = [get_app_json(ApplicationBase.wrap(app)) for app in apps if app]
 
         else:
-            apps = get_brief_apps_in_domain(domain)
+            # big TODO: write a new apps view for Formplayer, can likely cut most out now
```

Comment:
👍 don't think this'll be too hard and will cut down on ways we can break regular cloudcare by modifying formplayer